internal/vm/ioproxy.go

```diff
@@ -129,7 +130,11 @@ func (connectorPair *IOConnectorPair) proxy(
 
 		_, err := io.CopyBuffer(writer, reader, make([]byte, internal.DefaultBufferSize))
 		if err != nil {
-			logger.WithError(err).Error("error copying io")
+			if strings.Contains(err.Error(), "use of closed network connection") {
```

Comment:
Another error we see logged sometimes when copying to/from a FIFO is:
```
time="2019-12-13T16:22:10.889084273Z" level=error msg="error copying io" ExecID= TaskID=TestShimExitsUponContainerDelete_Isolated-1576254128261884676 error="read /proc/self/fd/9: file already closed" runtime=aws.firecracker stream=stdin vmID=b5b59453-6630-4a72-96b7-e0d7a93600a7
```

Can we also handle that case similarly to `use of closed network connection`? Maybe just checking `file already closed` would be sufficient.